src/youwol/app/environment/models/models_features.py

```diff
@@ -0,0 +1,7 @@
+# standard library
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True, kw_only=True)
+class Features:
+    config_dependant_browser_caching: bool = False
```

Comment:
In configuration's models we use camelCase for attribute's names.

Also, if the class need to be exposed in the API documentation, docstring is needed for the class and the attribute.